modules/tinymce/src/themes/silver/main/ts/ui/core/color/ColorSwatch.ts

```diff
@@ -83,13 +84,15 @@ const getAdditionalColors = (hasCustom: boolean): Menu.ChoiceMenuItemSpec[] => {
 const applyColor = (editor: Editor, format: ColorFormat, value: string, onChoice: (v: string) => void) => {
   if (value === 'custom') {
     const dialog = colorPickerDialog(editor);
+    const cssRgbValue = Css.get(SugarElement.fromDom(editor.selection.getStart()), format === 'hilitecolor' ? 'background-color' : 'color');
```

Comment:
Why not use the current `getCurrentColor` function above?

Reply:
Because for some reason I'd mentally filed it as checking the memory in options instead of the placement of the cursor. I'll update it, and use it.